### ISSUE TITLE
Typescript emitted no output for C:\Dev\Test\react-test\node_modules\react-arcgis\index.ts.

### ISSUE THREAD

**Issue by ebbypeter** [nicksenger/react-arcgis#38]
Module build failed when using react-arcgis 

```
ERROR in ./node_modules/react-arcgis/index.ts
Module build failed: Error: Typescript emitted no output for C:\Dev\Test\react-test\node_modules\react-arcgis\index.ts.
You should not need to recompile .ts files in node_modules.
Please contact the package author to advise them to use --declaration --outDir.
More https://github.com/Microsoft/TypeScript/issues/12358
    at successLoader (C:\Dev\Test\react-test\node_modules\ts-loader\dist\index.js:47:15)
    at Object.loader (C:\Dev\Test\react-test\node_modules\ts-loader\dist\index.js:29:12)
 @ ./src/components/well/well.tsx 14:21-44
 @ ./src/app.tsx
```
My package dependencies are as follows:
```
 "devDependencies": {
    "source-map-loader": "^0.2.3",
    "ts-loader": "^3.2.0",
    "typescript": "^2.6.2",
    "webpack": "^3.10.0"
  },
  "dependencies": {
    "@types/react": "^16.0.34",
    "@types/react-dom": "^16.0.3",
    "react": "^16.2.0",
    "react-dom": "^16.2.0"
  }
```

**Comment by nicksenger** [comment]
Hmm, it looks like typescript doesn't like the way this lib is being distributed. I'll be switching this package to be distributed as a few different bundles (ESM, UMD, etc) alongside the type definitions soon, which should take care of this.

Does using a previous version (3.0.0) fix this for you?

**Comment by nicksenger** [comment]
@ebbypeter this should be fixed in the latest release (3.1.3). Please reopen or create a new issue if you're still having problems.
